apps/prairielearn/src/ee/pages/instructorAiGenerateJobReview/instructorAiGenerateJobReview.html.ts

```diff
@@ -0,0 +1,70 @@
+import { formatDate } from '@prairielearn/formatter';
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { JobStatus } from '../../../components/JobStatus.html.js';
+import { Job } from '../../../lib/db-types.js';
+
+export function AiGenerateJobReviewPage({
+  resLocals,
+  genJobs,
+}: {
+  resLocals: Record<string, any>;
+  genJobs: Job[];
+}) {
+  return html` <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals })}
+      </head>
+      <body hx-ext="loading-states">
+        ${renderEjs(import.meta.url, "<%- include('../../../pages/partials/navbar'); %>", {
+          navPage: 'course_admin',
+          ...resLocals,
+        })}
+        <main id="content" class="container-fluid">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Generation job history</div>
+            <div class="table-responsive">
+              <table class="table table-sm table-hover">
+                <thead>
+                  <tr>
+                    <th>Number</th>
+                    <th>Date</th>
+                    <th>User</th>
+                    <th>Status</th>
+                    <th>Actions</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  ${genJobs.map(
+                    (job) => html`
+                      <tr>
+                        <td>${job.id}</td>
+                        <td>
+                          ${job.start_date == null
+                            ? html`&mdash;`
+                            : formatDate(job.start_date, resLocals.course.display_timezone)}
+                        </td>
+                        <td>${job.user_id ?? '(System)'}</td>
```

Comment:
A plain user ID isn't going to be useful to anyone; I'd recommend showing the user UID. This will require another join in your query.

Reply:
Changed in 9ec48833185d75e4ebfa4e4bb5a1682f21abe892.